components/kyma-environment-broker/internal/runtime/handler_test.go

```diff
@@ -262,3 +352,7 @@ func fixInstance(id string, t time.Time) internal.Instance {
 		Parameters:      internal.ProvisioningParameters{},
 	}
 }
+
+func fixRandomID() string {
+	return rand.String(16)
```

Comment:
nit:
in most cases a library `"github.com/google/uuid"` is used for generating UUIDs.